src/briefcase/commands/create.py

```diff
@@ -379,6 +379,40 @@ def install_app_support_package(self, app: BaseConfig):
             print()
             raise InvalidSupportPackage(support_package_url)
 
+    def build_env(self, app: BaseConfig):
+        """
+        Return an environment in which to build the dependencies for the app.
+        This should include the path to Python.h in CPPFLAGS, and libpython
+        and its path in LDFLAGS.
+
+        :param app: The config object for the app
+        """
+        env = dict(os.environ)
+        if self.platform == "android":
+            # sysconfig.get_config_var("INCLUDEPY")
+            includepy = os.path.join(
+                self.bundle_path(app), "app", "include",
+                "python{}m".format(self.python_version_tag),
+            )
+            assert os.path.exists(includepy), includepy
+            cppflags = env.get("CPPFLAGS", "").split(" ")
+            cppflags = [
+                "-I{}".format(includepy.replace(" ", "\\ ")),
+            ] + cppflags
+            env.update({"CPPFLAGS": " ".join(cppflags)})
+            # sysconfig.get_config_var("LIBDIR")
+            libdir = os.path.join(
+                self.bundle_path(app), "app", "libs", "arm64-v8a"
```

Comment:
Why is it hardcoded for arm64-v8a? It should be compatible with all available targets, as the emulator can run with a x86_64 arch for example.

Reply:
Yes, good point. This will require changes to the beeware/Python-Android-support project to include four architecture-specific "app_packages" directories, as well as changes to the beeware/briefcase-android-gradle-template project to add the right directory to sys.path. I'll try to implement this.